pkg/controllers/vdb/tlsservercertgen_reconciler.go

```diff
@@ -106,7 +109,7 @@ func (h *TLSServerCertGenReconciler) reconcileOneSecret(secretFieldName, secretN
 	if err != nil {
 		return err
 	}
-	h.Log.Info("created certificate and secret " + secret.Name)
+	h.Log.Info("created certificate and secret " + secret.Name + " for " + secretFieldName)
```

Comment:
```suggestion
	h.Log.Info("created certificate and secret", "secret", secret.Name, "field", secretFieldName)
```
Could you use this format for logging? logr encourages structured logging, which means using key-value pairs.

Reply:
structured logging is hard to read. I prefer one line format.

Reply:
Then use fmt.Sprintf() for those cases where you want to keep the text structure.